tests/test_validate_json.py

```diff
@@ -83,3 +83,23 @@ def test_table_name_is_correct(filename):
     basename = os.path.basename(filename)
     expected_table_name = basename.replace('.json', '')
     assert expected_table_name == table_name
+
+
+@pytest.mark.parametrize("filename", all_files)
+def test_no_empty_event_input_names(filename):
+    json_data = load_json_file(filename)
+    abi = json_data["parser"]["abi"]
+    for i, _input in enumerate(abi["inputs"]):
+        assert _input["name"] != "", \
```

Comment:
Does not catch the condition where `"name": null` in the json.

Suggestion:
```
assert _input["name"] not in (None, ""), \
```

Likewise line 103

Reply:
My bad, I had another look and realised we already use jsonschema for validation, which is probably a better way to this to begin with.